bootstrap/bootstrap.py

```diff
@@ -25,7 +25,9 @@
 
 from optparse import OptionParser
 
-tmpeggs = tempfile.mkdtemp()
+__version__ = '2.2.6'  # of bootstrap.py itself
+
+tmpeggs = tempfile.mkdtemp(prefix='bootstrap-')
```

Comment:
This is an unrelated change, but quite a nice one.

I've included it in #248, where it sits next to other bootstrap improvements.